packages/react/src/atomic/FileUploader/CHANGELOG.md

```
@@ -2,6 +2,13 @@
 
 The File Uploader component is used to upload files.
 
+## 2025-11-11 `2.7.0`
+
+#### 🎉 New features
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)